tests/Test-DbaTempDbConfiguration.Tests.ps1

```diff
@@ -27,27 +27,33 @@ Describe "$CommandName Unit Tests" -Tag 'UnitTests' {
         It "Should only contain $paramCount parameters" {
             $params.Count - $defaultParamCount | Should Be $paramCount
         }
+        It "Should throw on an invalid SQL Connection" {
+            Mock -ModuleName 'dbatools' Connect-SqlInstance { throw }
+            { Test-DbaTempDbConfiguration -SqlInstance 'MadeUpServer' -EnableException } | Should Throw
```

Comment:
Thanks so much for your updates! This one in particular is no longer needed as we test the internal command. It also adds 15 seconds to each test. Will merge upon removal 💯

Reply:
With the mock Connect-SqlInstance in there it shouldn't add any time to the tests. I did some comparisons on doing try/catch around the Connect-SqlInstance and the behavior of the function did change if you don't do the try/catch. I feel like it's a valid test to include to make sure that we are getting consistent behavior between cmdlets if an instance is unreachable.

Reply:
i suck with mocks - can you test the timing of the two and see? 

genuine question and up for a change of mind: if the connect-sqlinstance code block is universal, shouldn't that cover consistency across the board? did this particular command vary from the template?

Reply:
@potatoqualitee we do this in other test as well, but this is one area that we will be changing because we will no longer need to test the `Connect-SqlInstance` call in every test. Once we get the test for Connect-SqlInstance at 💯 we can remove mocks and all against it in all test.